src/Raven.Server/Documents/TombstoneCleaner.cs

```diff
@@ -182,10 +182,15 @@ private static long GetTombstoneDataForCollection(
 
         private void UpdateNotifications(List<BlockingTombstoneDetails> detailsSet)
         {
+            var key = AlertRaised.GetKey(AlertType.BlockingTombstones, nameof(AlertType.BlockingTombstones));
+
+            if (detailsSet.Count == 0 && _documentDatabase.NotificationCenter.Exists(key) == false)
+                return;
+
             if (detailsSet.Count > 0)
                 _documentDatabase.NotificationCenter.TombstoneNotifications.Add(detailsSet);
             else
-                _documentDatabase.NotificationCenter.Dismiss(AlertRaised.GetKey(AlertType.BlockingTombstones, nameof(AlertType.BlockingTombstones)));
+                _documentDatabase.NotificationCenter.Dismiss(key);
```

Comment:
Let's add the check inside the `Dismiss`.
We also send a notification when the item is dismissed.

Reply:
Done